src/lib/OpenEXR/ImfDwaCompressor.cpp

```diff
@@ -2250,10 +2251,12 @@ DwaCompressor::uncompress (
     // Flip the counters from XDR to NATIVE
     //
 
+    std::array<uint64_t, NUM_SIZES_SINGLE> counterBuf;
+    memcpy (counterBuf.data (), inPtr, counterBuf.size() * sizeof (uint64_t));
```

Comment:
I imagine since the data is all going to be touched subsequently, this has no performance implication, and may improve things through cache prewarming?

Reply:
It's a fairly small buffer, currently NUM_SIZES_SINGLE = 11, so 88 bytes?